includes/options/class-amp-options-menu.php

```diff
@@ -670,6 +681,88 @@ public function render_stories_export() {
 		<?php
 	}
 
+	/**
+	 * Render the stories settings section.
+	 *
+	 * @since 1.3
+	 */
+	public function render_stories_settings() {
+		$this->render_stories_settings_page_advance();
+	}
+
+	/**
+	 * Render the page advancement settings section.
+	 *
+	 * @since 1.3
+	 */
+	private function render_stories_settings_page_advance() {
+		$stories_settings_definitions = AMP_Story_Post_Type::get_stories_settings_meta_definitions();
+
+		/**
+		 * Currently the list of options with values and descriptions is hardcoded in PHP and JS.
+		 * There should be a single source for this data to be shared in the application.
+		 *
+		 * @todo Figure out the best way to keep these settings DRY.
+		 */
+		$auto_advance_after_options = $stories_settings_definitions['stories_settings_auto_advance_after']['data']['options'];
+
+		$auto_advance_after          = AMP_Options_Manager::get_option( 'stories_settings_auto_advance_after', '' );
+		$auto_advance_after_duration = AMP_Options_Manager::get_option( 'stories_settings_auto_advance_after_duration', 1 );
+		?>
+		<fieldset <?php disabled( ! current_user_can( 'publish_posts' ) ); ?>>
```

Comment:
I'd not print these settings in the first place when the user cannot edit them.

To see the settings screen one needs the `manage_options` cap anyway, no? So there is not much value in this check here.